### DISCUSSION TITLE
"Default PostCSS plugins are deprecated", but I don't use PostCSS

### DISCUSSION THREAD

**Discussion by Peeja** [storybookjs/storybook#18041] [Help]
On `@storybook/react v6.5.0-beta.0`, I'm getting:

```
(node:28843) DeprecationWarning: Default PostCSS plugins are deprecated. When switching to '@storybook/addon-postcss',
you will need to add your own plugins, such as 'postcss-flexbugs-fixes' and 'autoprefixer'.

See https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-default-postcss-plugins for details.
```

But I don't use PostCSS, or any CSS loading at all; I use Emotion for styling. According to #13669, it sounds like the only way to remove the warning is to add `addon-postcss` unnecessarily, disable _all_ depreciation warnings with `--no-deprecation`, or wait for 7.0 to come out. Is there a better way to make it go away?

**Comment by ianfp** [comment]
The MIGRATION notes explain how to suppress this error if you're not using PostCSS:
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-implicit-postcss-loader

**Comment by shilman** [comment]
I believe this has been fixed several years ago in Storybook 7. Closing as outdated.
